common/workunit/workunit.cpp

```diff
@@ -284,6 +296,13 @@ class CConstGraphProgress : public CInterface, implements IConstWUGraphProgress
                 tag = "node";
                 id += strlen(SubGraphScopePrefix);
                 break;
+            case SSTworkflow:
```

Comment:
I'm not sure either of these are needed.

Reply:
The statistic collection contains wfid.  Without this SSTworkflow case, the default case would be executed which throws an exception.  SSTchildgraph and SSTgraph has been added for the same reason.